go.mod

```diff
@@ -5,18 +5,18 @@ go 1.21
 toolchain go1.21.5
```

Comment:
Kubernetes v1.29.3 is built with go 1.21.8.
Shall we bump the toolchain to that version? I guess we'd better do it, but no big deal.

Reply:
agree